server/region.go

```diff
@@ -119,6 +119,15 @@ func (r *regionInfo) GetFollowers() map[uint64]*metapb.Peer {
 	return followers
 }
 
+func (r *regionInfo) GetFollower() *metapb.Peer {
```

Comment:
I think it does not worth introducing complexity. The balancer always tries to move a follower first, moving a leader typically happens in a single replica config, in this case, we are not able to find another peer to transfer leadership.